### DISCUSSION TITLE
IP assignment frustrations (I am officially stupid :))

### DISCUSSION THREAD

**Discussion by jms234** [netbox-community/netbox#13282] [Q&A] [answered]
Hi all

We have an existing IPAM called Gestio and I have exported the prefixes/IPs and imported them into Netbox.  The IPs are not assigned to any device interfaces in Netbox and for the majority of them that will always be the case, we just need a record of IP to server etc. 

However for the switches I do want to assign IPs to interfaces ie. L3 physical interfaces and more importantly the SVIs for the switch management. We have one management vlan (25) using 172.16.5.0/24 as the subnet. The IPs are in there but I cannot for the life of me work out how to assign them to the interfaces. 

If I edit the vlan 25 SVI there is no option to add an IP address that I can see so I wondered if it was because the IP was already there.  As a test I deleted the IPs and then added a new one and in the "Interface Assignment" window I could filter on "vlan25" which worked but it simply displays all the vlan25s it finds ie. 20 switches = 20 vlan25s and you have no way of knowing which one is tied to which device. 

I am obviously missing something very basic but how exactly do you tie an IP to a physical interface in the web UI.

**Comment by abhi1693** [comment, marked as answer]
Go to the interface, scroll down and click on the Add IP Address button. The default page that opens up is to create a new IP address. But you can also assign an existing one, to do so, click on the Assign tab on the same page

**Comment by jms234** [comment]
Ahh, that's it :). I was clicking on the edit interface and not actually just scrolling down to see all the other options. 

Many thanks for that.
